spec/controllers/entities_controller_spec.rb

```diff
@@ -62,22 +69,55 @@
         Rails.application.config.x.enable.entity_page = true
       end
 
-      it 'returns http success' do
-        get :show, locale: 'en', type: 'agent', namespace: 'base', identifier: '1234'
+      context 'without slug in URL' do
+        it 'redirects to URL with slug' do
+          allow(subject).to receive(:authorize!) { true }
```

Comment:
anywhere within these entity controller specs this authorize method is stubbed, it's 'legacy' code from when we were hiding entity pages via authorization.
These aren't being used now that we enable entity pages via the env variable.
Could you remove the 4 instances of this?